testing/regress/ecl/aggidx1.ecl

```diff
@@ -24,6 +24,8 @@ useSequential := #IFDEFINED(root.useSequential, false);
 
 //--- end of version configuration ---
 
+#option ('obfuscateOutput', true);
```

Comment:
I don't think we should be reusing tests. aggidx1.ecl should be copied and then the new code added.